frontend/locales/en.json

```
@@ -241,6 +241,7 @@
         "remove_title": "Remove Groups",
         "search_placeholder": "Name, Id",
         "sub_title": "Add or remove validator groups for {dashboardName}",
+        "this_dashboard": "this dashboard",
```

to align with the manage validators modal
```suggestion
        "this_dashboard": "your dashboard",
```